src/components/templates/Jazzbar/JazzTab/JazzTab.tsx

```diff
@@ -194,7 +195,10 @@ const Jazz: React.FC<JazzProps> = ({ setUserList, venue }) => {
                     key="main-event"
                     title="main event"
                     className="iframe-video"
-                    src={`${iframeUrl}?autoplay=1`}
+                    src={`${convertToEmbeddableUrl({
+                      url: venue.iframeUrl,
+                      autoPlay: venue?.autoPlay,
+                    })}`}
```

Comment:
`convertToEmbeddableUrl` should already return a string, no? In case it doesn't, adding ?? "" after it should serve the same as having those extra braces for the template string
```suggestion
                    src={ convertToEmbeddableUrl({
                      url: venue.iframeUrl,
                      autoPlay: venue?.autoPlay,
                    }) ?? "" }
```